### ISSUE TITLE
Main pages have no keyboard access

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#348]
All the main pages are not accessible from the keyboard.  We should add that.

**Comment by tfiduccia** [comment]
Pages are accessible now, but you cannot tell because there is no highlight when I tab to them.  Add some sort of visual cue.

**Comment by vincent99** [comment]
The default blue ring was restored a while ago so you should be able to tell what's focused now..

**Comment by tfiduccia** [comment]
Environment folder menu option is not visible (missing blue ring)
Cow, User Profile/Theme drop-down, download CLI are not part of the tabbing order

**Comment by lvuch** [comment]
resolved in https://github.com/rancher/ui/pull/511

**Comment by tfiduccia** [comment]
Version - master 3/8
Verified fixed
